### PR TITLE
[NUI] Fix CA2000 warning issues of GLWindow / ViewInternal.

### PR DESCRIPTION
**Pull request by AchoWang**
### Description of Change ###
Fix CA2000 warning issues of GLWindow / ViewInternal.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by huayongxu — APPROVED** [review]
lgtm.

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs [hunk 1]
```diff
@@ -70,7 +70,9 @@ internal float WorldPositionX
             get
             {
                 float temp = 0.0f;
-                GetProperty(View.Property.WORLD_POSITION_X).Get(out temp);
+                PropertyValue _wordPositionX = GetProperty(View.Property.WORLD_POSITION_X);
+                _wordPositionX?.Get(out temp);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs:74]: could you check this NUI ground rule suggestion #2 ? (https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule)
How do you think about using camel case for private field's naming.

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs [hunk 2]
```diff
@@ -80,7 +82,9 @@ internal float WorldPositionY
             get
             {
                 float temp = 0.0f;
-                GetProperty(View.Property.WORLD_POSITION_Y).Get(out temp);
+                PropertyValue _wordPositionY = GetProperty(View.Property.WORLD_POSITION_Y);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs:85]: could you check this NUI ground rule suggestion #2 ? (https://code.sec.samsung.net/confluence/display/GFX/NUI+Ground+Rule)
How do you think about using camel case for private field's naming.

File: src/Tizen.NUI/src/public/GLWindow.cs [hunk 3]
```diff
@@ -111,17 +111,24 @@ public Size2D WindowSize
...
                 Size2D ret = new Size2D(val.Width, val.Height);
+                val.Dispose();
 
                 return ret;
             }
             set
             {
-                var val = new Rectangle(Interop.GLWindow.GlWindow_GetPositionSize(swigCPtr), true);
-                Rectangle ret = new Rectangle(val.X, val.Y, value.Width, value.Height );
+                if (value == null)
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/GLWindow.cs:123]: could you add exception comment ? 
<exception cref=xxx> xxx </exception>
(example) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

File: src/Tizen.NUI/src/public/GLWindow.cs [hunk 4]
```diff
@@ -392,13 +401,18 @@ public GLWindow.GLWindowOrientation GetCurrentOrientation()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public void SetAvailableOrientations(List<GLWindow.GLWindowOrientation> orientations)
         {
+            if (orientations == null)
+                throw new ArgumentNullException(nameof(orientations));  // To fix CA1062 issue.
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/GLWindow.cs:405]: could you add exception comment ? 
<exception cref=xxx> xxx </exception>
(example) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs [hunk 5]
```diff
@@ -162,12 +168,16 @@ internal float ParentOriginX
...
-                GetProperty(View.Property.PARENT_ORIGIN_X).Get(out temp);
+                PropertyValue _parentOriginX = GetProperty(View.Property.PARENT_ORIGIN_X);
+                _parentOriginX?.Get(out temp);
+                _parentOriginX?.Dispose();
                 return temp;
             }
             set
             {
-                SetProperty(View.Property.PARENT_ORIGIN_X, new Tizen.NUI.PropertyValue(value));
+                PropertyValue pVal = new Tizen.NUI.PropertyValue(value);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs:178]: How about change "pVal" as "targetValue" or "setValue" ?

https://readlearncode.com/code-and-stuff/clean-code-variables/
: "Use words that can be pronounced and avoid abbreviations"

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs [hunk 6]
```diff
@@ -177,12 +187,16 @@ internal float ParentOriginY
             get
             {
                 float temp = 0.0f;
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs:189]: How about change "temp" as "temporary" or "returnValue" ?

https://readlearncode.com/code-and-stuff/clean-code-variables/
: "Use words that can be pronounced and avoid abbreviations"

File: src/Tizen.NUI/src/public/GLWindow.cs (hunk 4 above)
Comment (dasg34) [inline src/Tizen.NUI/src/public/GLWindow.cs:405]: This is not related to this commit(Fix CA2000 warning issues of GLWindow / ViewInternal.`).
Please separate this into different commit or PR for atomic commits.

**Review by dasg34 — CHANGES_REQUESTED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs (hunk 1 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs]: Done

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs (hunk 2 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs]: Done

File: src/Tizen.NUI/src/public/GLWindow.cs (hunk 3 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/GLWindow.cs]: Done
About warning CA1062, follow the comment of Mr. YeongJong Lee, remove it from current patch.

File: src/Tizen.NUI/src/public/GLWindow.cs (hunk 4 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/GLWindow.cs]: Done
About warning CA1062, follow the comment of Mr. YeongJong Lee, remove it from current patch.

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs (hunk 5 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs]: Done

File: src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs (hunk 6 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs]: Done

File: src/Tizen.NUI/src/public/GLWindow.cs (hunk 4 above)
Comment (AchoWang) [reply src/Tizen.NUI/src/public/GLWindow.cs]: Done
About warning CA1062, remove it from current patch.

**Review by dongsug-song — APPROVED** [review]

**Comment by dongsug-song** [comment]
@dasg34 
Hello, please review again so that this can be merged.
thank you.

**Review by dasg34 — APPROVED** [review]
